distribution/pom.xml

```diff
@@ -242,6 +242,58 @@
 
         <plugins>
             <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>license-maven-plugin</artifactId>
+                <configuration>
+                    <outputDirectory>target</outputDirectory>
+                    <thirdPartyFilename>NOTICE.txt</thirdPartyFilename>
+                    <acceptPomPackaging>true</acceptPomPackaging>
+                    <fileTemplate>org/kaazing/community/notice/third-party-template.ftl</fileTemplate>
+                    <useMissingFile>true</useMissingFile>
+                    <missingFile>src/license/THIRD-PARTY.properties</missingFile>
+                    <failIfWarning>true</failIfWarning>
+                </configuration>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>add-third-party</goal>
+                        </goals>
+                        <phase>generate-resources</phase>
+                   </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <version>1.4.1</version>
+                <dependencies>
+                    <dependency>
+                        <groupId>org.kaazing</groupId>
+                        <artifactId>community.enforcer</artifactId>
+                        <version>develop-SNAPSHOT</version>
+                    </dependency>
+                </dependencies>
+                <executions>
+                    <execution>
+                        <id>default-enforce</id>
+                        <configuration combine.self="override">
+                            <skip>false</skip>
+                            <rules>
+                                <myCustomRule implementation="org.kaazing.community.enforcer.rule.RequireSameFile">
+                                    <originalFilePath>src/main/gateway/NOTICE.txt</originalFilePath>
+                                    <revisedFilePath>target/NOTICE.txt</revisedFilePath>
```

Comment:
The rule is implemented to only take the relative path to the current project. 
I created [PR](https://github.com/kaazing/community/pull/45) in the community repo with this change. After this is merged and community released, then I can update the paths in the pom to use mvn properties,